test/modules/sdk/objections/service.spec.unit.ts

```diff
@@ -160,6 +160,19 @@ describe("objections SDK service unit tests", () => {
       HTTP_DELETE,
     );
   });
+
+  it("should call objections API getting an objection", () => {
+    objectionsSdk.getObjection(COMPANY_NUMBER,
```

Comment:
i think you should check that the function returns the object as expected and call with await (make this test function async)

Reply:
Done.